core/src/test/scala/org/broadinstitute/dsde/rawls/mock/MockWorkspaceManagerDAO.scala

```diff
@@ -23,7 +28,8 @@ class MockWorkspaceManagerDAO(
 
   val references: TrieMap[(UUID, UUID), DataRepoSnapshotResource] = TrieMap()
 
-  def mockGetWorkspaceResponse(workspaceId: UUID) = new WorkspaceDescription().id(workspaceId)
+  def mockGetWorkspaceResponse(workspaceId: UUID) =
+    new WorkspaceDescription().id(workspaceId).gcpContext(new GcpContext().projectId("some-project-id"))
```

Comment:
Now that we actively check the snapshot -> dataset -> cloudPlatform, this object needs to be adequately hydrated for all the existing tests to continue passing.